lib/client/client.wilson.js

```diff
@@ -27,6 +27,9 @@
   var _controllerProvider = null;
   var _provider           = null;
 
+  // Decorate Lodash with conveniences
```

Comment:
Since that special classify method just re-uses lodash string methods to re-create the classify functionality, this should be safe here (just not as pretty)